<!DOCTYPE html>
<html lang="en">
    <head>
	    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <title>Do Work!</title>
        <meta name="description" content="Ryan Silva's Lab Work">
        <meta name="author" content="Ryan Silva, Branched from Todd Branchflower">
        <link rel='stylesheet' type='text/css' href='//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.min.css'>
        <script src='//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/js/bootstrap.min.js'></script>
	<script type="text/x-mathjax-config">
		  MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'],
		                  ['\\(','\\)']]}});
	 </script>
	<script type="text/javascript"
		   src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
         </script>
	<link rel="stylesheet" href="http://yandex.st/highlightjs/8.0/styles/default.min.css">
	<script src="/node_modules/highlight.js/lib/index.js"></script>
    </head>
    <body>
        <div class = "container">
            <div class = "navbar navbar-inverse">
                <div class = "navbar-inner">
                    <a class = "brand" href="/index.html">AVLIS</a>
                    <ul class = "nav">
                        <li><a href="/">Home</a></li>
                        <li><a href="/setup/index.html">Setup</a></li>
                        <li><a href="/equipment/index.html">Equipment List</a></li>
                        <li><a href="/lab/index.html">Lab Work</a></li>
                        <li><a href="/links/index.html">Links</a></li>
                        <li><a href="/papers/index.html">Papers</a></li>
                        <li><a href="/TeamDocs/index.html">Team Docs</a></li>
                        <li><a href="/action_items/index.html">Action Items</a></li>
                    </ul>
                </div>
            </div>
            <h1>Biosensors</h1>
<ul>
<li><a href="http://pubs.acs.org/doi/full/10.1021/ac9805201">Biosensor for Direct Determination of Organophosphate Nerve Agents using Recombinant E.Coli</a></li>
</ul>
<p><img src="/papers/experimental_setup.gif" alt="Experimental Setup"></p>
<ul>
<li><p><a href="/papers/ApplicationsSynBio.pdf">Synthetic Biology and the Art of BioSensor Design</a></p>
</li>
<li><p><a href="/papers/pH_based_biosensor_arsenic_drinkingwater.pdf">pH-based biosensor for detection of arsenic in drinking water</a></p>
</li>
</ul>
<p><img src="pHResponse_pH6to8.png" alt="pH Response"></p>
<ul>
<li><a href="/papers/Edinburgh_Development_arsenic_drinking_water.pdf">Development of a nir fovel biosensor for the detection of arsenic in drinking water</a><ul>
<li>World Health Organization recommended limit is 10 ppb or <a href="http://www.who.int/mediacentre/factsheets/fs372/en/">10 $\mu$g/litre</a></li>
<li>Sensor can detect 5 ppb, albeit with a caveat<ul>
<li>You must wait 500 minutes before the Output Dynamic Range between 0ppb and 5ppb goes above .1pH</li>
</ul>
</li>
<li>pH vs Time:</li>
</ul>
</li>
</ul>
<p><img src="pHresponse.png" alt="pH Response"></p>
<ul>
<li><a href="/papers/NanowireFET_Nanotechnology.pdf">Optimization of pH sensing using silicon nanowire field effect transistors with HfO2 as the sensing surface</a></li>
</ul>
<p><img src="IV.png" alt="IV Curve"></p>
<ul>
<li><p><a href="/papers/Engineering_EColi_tosee_light.pdf">Engineering E Coli to see light</a></p>
</li>
<li><p><a href="/papers/Synthetic_edge_Detection.pdf">A Synthetic Genetic Edge Detection Program</a></p>
</li>
<li><p><a href="/papers/Spatiotemporal.pdf">Spatiotemporal control of cell signalling using a light-switchable protein interaction</a></p>
</li>
<li><p><a href="/papers/nrm3767.pdf">Realizing the potential of synthetic biology</a></p>
</li>
<li><p><a href="/papers/berezamalcolm2014.pdf">Environmental Sensing of Heavy Metals Through Whole Cell Microbial Biosensors: A Synthetic Biology Approach</a></p>
<ul>
<li>Refs 13 (aleksic2007), 47 (demora2011), 50 (stocker2003) and 55 (joe2012) all make sensors that modulate pH</li>
</ul>
</li>
<li><p><a href="/papers/uBioRobots.pdf">Sensors for Micro Bio Robots via Synthetic Biology</a></p>
</li>
<li><p><a href="/papers/Mammalian_pHSensor.pdf">A Synthetic Multifunctional Mammalian pH Sensor and CO2 Transgene-Control  Device</a></p>
<ul>
<li>Device pitched as a treatment for diabetes<blockquote>
<p>Life operates...within a narrow biochemically permissive range (e.g., pH<br>7.35-7.45 in humans</p>
</blockquote>
</li>
</ul>
</li>
<li><p><a href="/papers/Tuning.pdf">Tuning Response Curves for Synthetic Biology</a></p>
<blockquote>
<p>Synthetic biology includes a concerted effort to formalize an engineering discipline suitable for the design and implementation of novel biological systems.</p>
</blockquote>
<ul>
<li>Biology operates in a noisy environment</li>
<li><p>Response curve defined by a single first order differential equation, which is a highly approximate model based on &quot;empirical observations&quot; </p>
</li>
<li><p>Hill Function</p>
</li>
</ul>
</li>
</ul>
<p>$$\theta(x)=\frac{x^n}{K^n+x^n}$$</p>
<ul>
<li>$x$ = unbound ligand concentration</li>
<li>$\theta(x)$ = avg fraction of binding sites occupied by an input ligand as a function of unbound ligand concentration.</li>
<li>$K$ = The Hill constant. Rough indicator of the level of ligand concentrations needed to induce saturation ($x \gg K$)</li>
<li>$n$ = The Hill coefficient. <strong>The larger the value of $n$, the steeper the slope of the Hill function</strong><ul>
<li>$n &gt; 1$ indicates cooperativity, where affinity increases in the presence of previously bound ligands</li>
<li>$0 &lt; n &lt; 1$ indicates negative cooperativity, where affinity is reduced</li>
<li>$n = 1$ indicates a noncooperative reaction</li>
</ul>
</li>
</ul>
<p><img src="Tuning_Fig2.png" alt="Figure 2"></p>
<ul>
<li>Equation for Protein Expression</li>
</ul>
<p>$$\frac{dy}{dt}=k&#39; + k \left(\frac{x^n}{K^n+x^n} \right)$$</p>
<pre><code>-<span class="ruby"> <span class="hljs-variable">$y</span><span class="hljs-variable">$ </span>is the concentration of the protein being expressed
</span>-<span class="ruby"> <span class="hljs-variable">$k</span><span class="hljs-string">'$ is the basal rate of production
</span></span>-<span class="ruby"> <span class="hljs-variable">$k</span><span class="hljs-variable">$ </span>is the maximum additional production rate arising from up-regulation
</span>-<span class="ruby"> <span class="hljs-variable">$\</span>left(\frac{x^n}{<span class="hljs-constant">K</span>^n+x^n} \right)<span class="hljs-variable">$ </span>is the increasing sigmodal <span class="hljs-constant">Hill</span> function</span></code></pre>
<ul>
<li>Equation for Protein Repression</li>
</ul>
<p>$$\frac{dy}{dt}=k&#39;+k\left(\frac{K^n}{K^n+x^n}\right)$$</p>
<pre><code>-<span class="ruby"> <span class="hljs-variable">$k</span><span class="hljs-string">'+k$ is the basal expression rate
</span></span>  -<span class="ruby"> <span class="hljs-variable">$k</span><span class="hljs-string">'$ accounts for the fact that complete repression may not be possible
</span></span>-<span class="ruby"> <span class="hljs-variable">$\</span>left(\frac{<span class="hljs-constant">K</span>^n}{<span class="hljs-constant">K</span>^n+x^n}\right)<span class="hljs-variable">$ </span>is the decreasing sigmodal <span class="hljs-constant">Hill</span> function</span></code></pre>
<ul>
<li>Types of Tuning</li>
</ul>
<p><img src="tuning_fig1.png" alt="Types of Tuning"></p>
<ul>
<li><strong>Vertical Scaling</strong> <ul>
<li>Change the promoter-gene copy number<ul>
<li>Multiplies the rate at which mRNA is produced and therefore translated</li>
</ul>
</li>
<li>Change RBS strength<ul>
<li>Codon optimization</li>
</ul>
</li>
</ul>
</li>
<li><strong>Vertical Shifting</strong><ul>
<li>Introduce or tune a constitutive (always on) source of output $y$</li>
</ul>
</li>
<li><strong>Vertical Extension</strong><ul>
<li>Up-Regulation: Tune the activation potency of bound TF protein </li>
<li>Down-Regulation: No known single-step method (excluding $k&#39;=0$ case, which is identical to vertical scaling)</li>
</ul>
</li>
<li><strong>Leakage</strong><ul>
<li>Up-Regulating: Leakage is that which is always active</li>
<li>No known direct method</li>
<li>Down-Regulating: Leakage is that which cannot be repressed</li>
<li>Vary the repression strength of each bound TF protein </li>
</ul>
</li>
<li><strong>Horizontal Scaling</strong><ul>
<li>Tune the Hill constant $K$ (increasing $K$ scales curve to the right)</li>
<li>$K$ is related to the effective binding affinity of the input signal to the process</li>
<li>Tune the binding affinity of the TF to the promoter</li>
</ul>
</li>
<li><strong>Steepness</strong><ul>
<li>Tune the Hill coefficient $n$ (increasing $n$ leads to increasing steepness)</li>
<li><em>Ultrasensitivity:</em> A biochemical process that has a steep or switch-like steady-state response curve</li>
<li>Requires adjusting the effective binding cooperativity</li>
<li>Implies the cooperative binding of multiple TFs to the same promoter</li>
</ul>
</li>
<li><strong>Dynamic Range</strong><ul>
<li>The observable space between uninduced and fully saturated induction</li>
</ul>
</li>
</ul>
<h1>Microfluidics</h1>
<ul>
<li><p><a href="/papers/Microfluidics_and_SynBio.pdf">Combining microfluidics and synthetic biology</a></p>
</li>
<li><p><a href="/papers/thorsen2002.pdf">Microfluidic Large-Scale Integration</a></p>
<ul>
<li>Review paper for the state of the microfluidic art in 2002<blockquote>
<p>A key component of these networks is the fluidic multiplexor, which is a combinatorial array of binary valve patterns that exponentially increases the processing power of a network by allowing complex fluid manipulations with a minimal number of inputs.</p>
</blockquote>
</li>
</ul>
</li>
</ul>
<p><img src="mux.png" alt="Microfluidic Multiplexor!"></p>
<blockquote>
<p>On the basis of the utility of these examples, we believe that other concepts developed for electronic integrated circuits can be usefully transferred to chemical and biochemical analysis and processing in microfluidic devices</p>
</blockquote>
<ul>
<li><p><a href="/papers/Abstraction_Layers.pdf">Abstraction Layers for Scalable Microfluidic Biocomputers</a></p>
</li>
<li><p><a href="/papers/soft_litho_CAD.pdf">CAD for Microfluidic Chips Based on Multilayer Soft Lithography</a></p>
<blockquote>
<p>A second difference between electronic CAD and microfulidic CAD is the fast turn-around time in manufacturing microfluidic chips.</p>
</blockquote>
<ul>
<li>I argue that prototyping using FPGAs is significantly faster than manufacturing microfluidic chips.</li>
</ul>
</li>
<li><p><a href="/papers/Recent_uF_Developments.pdf">Recent developments in microfluidic large scale integration</a></p>
<ul>
<li>A programmable uF platform has kind-of been investigated, albeit just for mixing and processing via fluorescent derivatization.</li>
</ul>
</li>
<li><p><a href="/papers/Programmable_uF_automaton.pdf">Digitally programmable microfluidic automaton for multiscale combinatorial mixing and sample processing</a></p>
<ul>
<li>The review paper quotes the shortcomings of the device as: <blockquote>
<p>larger hybrid PDMS/glass valve technology differentiates itself from mLSI and is more suitable for microliter scale fluid volue processing (the chamber volume is 120 nL); moreover this technology requires one external valve per chamber and it seems unlikely that any reduction is possible</p>
</blockquote>
</li>
<li>On second glance, an FPGA-like uF platform has been done...damn:</li>
</ul>
</li>
<li><p><a href="/papers/FPGA_uF.pdf">A software-programmable microfluidic device for automated biology</a></p>
<ul>
<li>And again...This time using a combination of flow based and electrowetting-enabled droplet based uF</li>
</ul>
</li>
<li><p><a href="/papers/Programmable_electrowetting_uF.pdf">A programmable droplet-based microfluidic device applied to a multiparameter analysis of single microbes and microbial communities</a></p>
<blockquote>
<p>This is largely due to the prevailing para-digm in microfluidic research in which devices areb hardwiredb  for specific fluid handling tasks, necessitating a customized design for each application or change in protocol. This application-spe-cific approach requires iterative cycles of device design, fabrica-tion, and testing, presenting a major obstacle to the development of new applications and limiting user adoption and community access. In analogy to how programmable integrated circuits en-abled a broader community of developers and nonexpert users,the advancement of programmable microfluidic devices standsto dramatically enhance the pervasiveness and impact of micro-fluidic systems (15)</p>
<ul>
<li><strong>Idea</strong>: what if you combine MHDL with an actual chip by building a Place-and-Router?<ul>
<li>or what if you built a programmable mLSI chip that could be addressed by MHDL, or a subset thereof?</li>
</ul>
</li>
<li><strong>This doesn&#39;t seem to have been investigated!</strong><ul>
<li>The electrowetting device uses a LABVIEW program to control the chip</li>
</ul>
</li>
</ul>
</blockquote>
</li>
<li><p><a href="Methods_ProgramElectrouF.pdf">Methods used to make device</a></p>
<ul>
<li>The claimed FPGA-like devices uses Visual Basic to address the solenoid elements</li>
<li>The evolution of the FPGA began with the HDL, not the other way around...right?</li>
</ul>
</li>
<li><p><a href="/papers/MHDL_Synth.pdf">Automatic Synthesis of Microfluidic Large Scale Integration Chips from a Domain-Specific Language</a></p>
<ul>
<li>MHDL is a specification language for mLSI chip architectures, as opposed to assay specification. An assay specification language compatible with MHDL is<br>Biocoder as shown in this paper: - Introduced here: </li>
</ul>
</li>
<li><p><a href="/papers/MHDL.pdf">Design and Verification Tools for Continuous Fluid Flow-based Microfluidic Devices</a></p>
</li>
</ul>
<h1>Characterization</h1>
<ul>
<li><p><a href="/papers/Tabor_OptoChar.pdf">Optogenetic characterization methods overcome key challenges in synthetic and systems biology</a></p>
<blockquote>
<p>Component signal processing properties, such as the relative amplitude (gain) and time delay of output relative to input for various signals, are captured using <strong>phenomenological</strong>, or &#39;black box&#39; mathematical models. The models are used to reliably assemble components into circuits with more advanced capabilities, such as band-pass filters or amplifiers, and eventually into complete systems, such as computers or radios</p>
</blockquote>
<ul>
<li>i.e. Derive models from experimentation, NOT FROM BIOLOGICAL THEORY!</li>
</ul>
</li>
</ul>
<blockquote>
<p>Given sufficient measurements, one should be able to formulate a mathematical model that accurately predicts receptor outputs for input signals that have not been tested. Because the only purpose of the model is to predict receptor I/O, a black-box model will typically be preferred over a more complicated version reflecting the receptor molecular biology. Once it is parameterized with experimental data, the model can be used to simulate the receptor response to a wide range of different inputs in silico, thus enabling the computational design of effector signals capable of driving biological signals with desired quantitative and temporal features and satisfying challenge two. Additionally, if receptor outputs can be measured using fluorescent proteins in real time, an in silico feedback controller can adjust the designed input signal to account for any deviations from the model prediction</p>
</blockquote>
<ul>
<li><a href="/papers/Evolution.pdf">Evolution Paper</a></li>
</ul>
<ul>
<li><a href="/papers/Analog_Synbio.pdf">Analog Synthetic Biology</a></li>
</ul>
<blockquote>
<p>Nevertheless, we must not forget that digital computation has not offered an effective paradigm for computing efficiently and precisely with noisy and unreliable devices; that multi-logic-gate computations can impose significant metabolic or toxicity burdens on cells owing to their need to use a lot of parts and power; that the fact that there are five to six orders of magnitude fewer genes per cell than digital transistors per chip means that using genes to only perform logic is likely not an efficient way to attain high complexity; that a library of b digital partsb  with good onb off ratios, low standby power consumption and low crosstalk does not exist in biology; that the computing basis functions in cells are not really logic functions and abstracting them as such compromises computational efficiency; and that logic basis functions are not the only universal computation primitives.</p>
</blockquote>
<ul>
<li>No static discipline? Don&#39;t just make one up that doesn&#39;t work the same way in cells as it does in computers!</li>
</ul>
<blockquote>
<p>The b 1b s and b 0b s of todayb s digital computers are useful abstractions of the analog signals in cells, but are often an oversimplification.</p>
</blockquote>
<ul>
<li>If digital abstraction can&#39;t be used to create a working system, then it isn&#39;t digital abstraction!</li>
</ul>
<blockquote>
<p>Furthermore, as in analog circuits, the wiring of the output of one circuit to the input of another leads to b loadingb  interactions that degrade overall function and prevents simple modular digital abstractions from being effective [13].</p>
</blockquote>
<ul>
<li>Figure 1b allows you to focus on a &quot;protein input protein output&quot; point of view<br><img src="/papers/analog.png" alt="analog"></li>
</ul>
<ul>
<li><a href="/papers/voigtNORGate.pdf">Voigt Nor Gate</a></li>
</ul>
<h1>Cell to Cell communication applications</h1>
<ul>
<li><p><a href="/papers/felix2013.pdf"> Spatial Structure Facilitates Cooperation in a Social Dilemma: Empirical Evidence from a Bacterial Community</a></p>
<ul>
<li>GASP = Growth Advantage in Stationary Phase<blockquote>
<p>The fact that cooperation is simultaneously ubiquitous and fragile, begs for an explanation regarding its emergence and maintenance.<br>E. coli bacteria collectively restrain growth to avoid population collapse when resources become scarce [21]b [23]. The collective decision to restrain growth before all resources are depleted is a form of altruistic cooperation, and one can think of the entry into stationary phase as a social contract: a collective rule of density-dependent growth regulation [9], [24]. The long-term benefits of going into stationary phase are evident: by limiting population growth, bacteria ensure that enough resources will be available to maintain a viable population for an extended period, and survive until conditions improve and growth can be resumed. In the short term however, b cheaterb  cells that ignore the collective decision to restrain growth and that utilize the scarce resources to proliferate instead of investing in maintenance, may increase in numbers. As a result, cheaters jeopardize the survival of the collective b  which in turn could face a tragedy of the commons [25].<br>In a Prisoner&#39;s Dilemma, the Sucker&#39;s payout is negative for their model (only look at the sign of the payoff differences) to work; in a Snowdrift regime the Sucker&#39;s payout is positive). </p>
</blockquote>
</li>
</ul>
</li>
<li><p><a href="/papers/Molina2014.pdf">Cooperation and bacterial pathogenicity: an approach to social evolution</a></p>
</li>
<li><p><a href="/papers/Goers2014.pdf">Co-culture systems and technologies: taking synthetic biology to the next level</a></p>
<ul>
<li>Review paper on experimental and theoretical co-culture set-ups in nature and synbio</li>
<li>Co-culture setups exist to study cell-cell interactions of any kind</li>
<li>Current co-culture systems (methods of separating cell populations<br><img src="Goers2014_F3.jpg" alt="Drawing" style="width: 500px;"/></li>
<li><strong>Does TweeColi offer a novel method of conducting co-culture experiments?</strong><blockquote>
<p>As population interactions suffer from context dependency and can change in different environments, characterization should include stress limits. For example, an engineered interaction was reported to be functional only within certain pH ranges [39]. Taking context dependency into account does not imply a departure from the engineering and computing analogy on which synthetic biology is based, but rather an improved appreciation of the challenges that come with the analogy. Just as civil engineers need to consider the environmental stresses that a building will have to withstand, synthetic biologists need to consider the working environment of bio-parts at the stage of circuit design. On the other hand, co-culture technologies can support the genetic engineering that is at the basis of synthetic systems. Genetic programming cannot entirely predict cellular behaviour, as environment plays a role and there are still limits in what can be controlled genetically [41]. Using genetic engineering and controlled environmental conditions through experimental set-up together could allow a level of population control that was previously thought impossible. Emergent and unpredicted behaviour can arise in co-cultures and may or may not be desired. One important question in the choice of cultivation technology is what properties of the natural populations are to be preserved or suppressed. Overall, populations can be controlled genetically and through the environment. Which aspects can be controlled genetically or through set-up is a subject of ongoing research.</p>
</blockquote>
</li>
</ul>
</li>
<li><p><a href="/papers/Bacchus2012.pdf">Engineering of synthetic intercellular communication systems</a></p>
<ul>
<li>Review paper of intercellular communication and multicellular consortia</li>
<li>Can we do the population control experiment in A below using TweeColi?<br><img src="/papers/Bacchus2012_F1.jpg" alt="Bacchus figure 1"></li>
</ul>
</li>
<li><p><a href="http://link.springer.com/article/10.1007/s00792-009-0261-3/fulltext.html#CR84">Cultivating the uncultured: limits, advances and future challenges</a>  <strong>No pdf link</strong></p>
<ul>
<li>Culture isolation using pH control?<br><img src="culture_isolation.gif" alt="culture_isolation.gif"></li>
</ul>
</li>
</ul>
<h1>Synthetic Biology Design</h1>
<ul>
<li><a href="/papers/Lewens2013.pdf">From <em>bricolage</em> to BioBricks: Synthetic Biology and rational design</a><ul>
<li>Explores the limitations of the synbio design process </li>
<li>Synthetic Biology defined by the European Commission as &quot;the synthesis of complex, biologically based (or inspired) systems which display functions that do not exist in nature. This engineering perspective may be applied at all levels of the hierarchy of biological structures...In essence, synthetic biology will enable the design of &#39;biological systems&#39; in a rational and systematic way&#39; <blockquote>
<p>The UK Health and Safety executive says that synthetic biology can be described as b the design and construction of new biological parts, devices and systems, and the redesign of existing, natural biological systems for useful purposesb  (HSE, 2007). The European Commission says that synthetic biology is b the engineering of biology: the synthesis of complex, biologically based (or inspired) systems which display functions that do not exist in nature. This engineering perspective may be applied at all levels of the hierarchy of biological structures b &amp; In essence, synthetic biology will enable the design of b biological systemsb  in a rational and systematic wayb  (European Commission, 2005). The UK Parliamentary Office of Science and Technology tells us that synthetic biology b describes research that combines biology with the principles of engineering to design and build standardised, interchangeable biological DNA building-blocks. These have specific functions and can be joined to create engineered biological parts, systems and, potentially, organisms. [Synthetic biology] may also involve modifying naturally occurring genomes b &amp; to make new systems or by using them in new contextsb  (POST, 2008).</p>
</blockquote>
</li>
<li>&#39;Bottom up&#39; design = construction of genetically specified components and combine them to create novel organic devices</li>
<li>Top down = begin with naturally occurring organisms and seek to remove redundant or unnecessary parts to produce simpler, less &#39;noisy&#39; organic machines.</li>
<li>Research Opportunity?<blockquote>
<p>the notion that an engineering design perspective characterises a variety of synthetic biology approaches remains in very recent research articles</p>
</blockquote>
</li>
<li>Irrational vs Rational design process<blockquote>
<p>It is nonetheless reasonable to place synthetic biology towards one end of a continuum between purely b blindb  processes of organic modification at one extreme, and wholly rational, design-led processes at the other. </p>
<ul>
<li>Irrational:<br>They are b irrationalb  only because the agent who oversees the process need not understand why the resultant entity works as it does. Whether a method is b rationalb  or b irrationalb  is itself something that comes by degrees: an engineer may not understand why the parts of a machine work as they do, but he may still be able to predict in rough-and-ready ways how they will behave when brought together. After several moderately successful attempts he may be surprised at the makeup of the machine that works best.</li>
</ul>
</blockquote>
</li>
<li>Nazi attempt to breed back aurochs failed because<blockquote>
<p>On the basis of vague criteria and without proper knowledge of the appearance of the aurochs, the two brothers made inaccurate selections among the crossbreeding products of various cattle breeds. They did not use the knowledge about the aurochs that was available at the time, nor did they take advantage of the breeding techniques others were using to create new cattle breeds in the same period. (Van Vuure, 2005, p. 366)<br><strong>Poor engineering design methodology</strong></p>
</blockquote>
</li>
<li>With regard to directed evolution, &#39;the creativity of synthetic biology itself continues to rely on processes whose efficacy may be inscrutable to the designers themselves, and which have much in common with familiar breeding techniques&#39;</li>
<li>Irrational design in &#39;evolutionary electronics&#39;<ul>
<li>An FPGA was altered slightly and continuously tested for an ability to do a task and the best design could never have been achieved using normal design techniques<blockquote>
<p>[These cells] cannot be clamped [i.e. returned to their default b blankb  state] without degrading performance, even though there is no connected path by which they could influence the output b &amp; They must be influencing the rest of the circuit by some means other than the normal cell-to-cell wires: this probably takes the form of a very localised interaction with immediately neighbouring components. Possible mechanisms include interaction through the power-supply wiring, or electromagnetic coupling. ( Thompson, 2002)</p>
</blockquote>
</li>
</ul>
</li>
<li>Reducible complexity is not an artifact of evolution (T. Lewens 2004), therefore we should not expect to isolate components and then stick them together in a different order and expect rational behavior<blockquote>
<p>Evolution has been free to explore the full repertoire of behaviours available from the silicon resources provided, even being able to exploit the subtle interactions between adjacent components that are not directly connected. (Thompson, 2002)</p>
</blockquote>
</li>
<li><strong>Does natural selection prefer modularity? Will it pick a modular design vs a non-modular design?</strong><blockquote>
<p>It remains unclear whether natural selection is able to favour modularity<br>Natural selection is not constrained by a prior methodological commitment to rationality, and since natural selection must build traits that are effective in the face of regular internal and external insults, natural selection also needs to ensure that its products are robust against a range of environmental perturbations</p>
</blockquote>
</li>
<li>Abandon the idea of modularity in synthetic biology?<blockquote>
<p>Biological complexity is here acknowledged, the barriers it presents to human intervention are explicitly admitted, and attempts are made to avoid it (in the first instance, at least) via the creation of simpler systems.</p>
</blockquote>
</li>
<li>Citing M. O&#39;Malley &#39;Exploration, iterativity and kludging in synthetic biology&#39;: <blockquote>
<p> Rather than focusing on hypotheses, testing and inference, synthetic biology invites attention to the practices of exploration, iterativity and kludging. Examining such strategies in relation to synthetic biology offers new avenues of insight for philosophy of science and biology.</p>
</blockquote>
</li>
<li><strong>Why not design with system complexity in mind? For SynBio, this comes at the cost of modularity</strong></li>
</ul>
</li>
</ul>
<ul>
<li><p><a href="/papers/Nordmann2014.pdf">Synthetic Biology at the Limits of Science</a></p>
<ul>
<li>How should we approach understanding biology for achieving &#39;technical control&#39;?<ul>
<li>Understanding</li>
<li>Rationality</li>
<li>Design</li>
</ul>
</li>
<li>Understanding<ul>
<li>Ability to create is touted by some as necessary to demonstrate understanding <ul>
<li>Hypothesize, model, verify through experimentation </li>
<li>Creating is a method of knowledge exploration?? The author seems dubious<blockquote>
<p>If one wants to engineer a biological structure or process, one &quot;cannot help&quot; but gain understanding also (Benner/Sismour 2005)</p>
</blockquote>
</li>
</ul>
</li>
<li>Richard Feynman: &quot;What I cannot create, I do not understand&quot; != &quot;What I can create, I do understand&quot;<blockquote>
<p>The ability to create now appears as a sufficient condition for understanding, suggesting that &quot;understanding&quot; can leap ahead of explanation and prediction</p>
</blockquote>
</li>
<li>Understanding involves removing complexity for the sake of deriving cause</li>
</ul>
</li>
<li><p>Rationality (Engineering principles)</p>
<ul>
<li><p>Why is design by trial-and-error (irrational design) seen as &quot;less civilized&quot; than rational design?</p>
<blockquote>
<p>Those still working in the mode of trial and error have not ascended as yet to a level of intellectual and technical control that would allow them to invent new processes or devices simply from considerations of theory and principle</p>
</blockquote>
<ul>
<li>counter example: &quot;rational drug design&quot; proved inferior to high-throughput, trial and error methods (Adam 2010) so the scientific community responds with nanomedicine and visions of targeted drug delivery (&quot;this time we&#39;ll get it right&quot;)</li>
</ul>
</li>
<li><p>Amazing questions:</p>
<blockquote>
<p>Similarly, the aspirations of synthetic biology are oft en judged in these terms Can the &#39;synthesis&#39; of biological structures or processes proceed in a planned, deliberate, theory - and evidence - based manner such that the intended outcome is sues as if from a blueprint (Giese et al. 2013, Ob Malley 2009 , Gramelsberger 2013 , Lewens 2013 ) ? And , if this is not the case , is this only b not yet b  the case, likely to become possible in just a few more years , or does it owe to a disc iplinary style of doing things b  with chemists seeking rational control while bioengineers are more comfortable with tinkering ? Or does the failure of rational design owe to a limit of biological complexity that is irreducible and thus an insurmountable limit of control?</p>
</blockquote>
</li>
<li><p>Author&#39;s assertion: &quot;The hierarchical conception that places rational engineering above tinkering is blind to the possibility of rational tinkering&quot;</p>
</li>
<li>Engineering may be opposed to &quot;blind groping&quot; but not search strategies that exploit random variations or iterative adaption and tuning.<blockquote>
<p>These, to be sure, are rational strategies by which to work around limits of knowledge, and to achieve technical solutions in the absence of information about mechanical detail. <strong>In other words, these are rational strategies to create robust black boxes or modules</strong></p>
</blockquote>
</li>
<li>Author&#39;s engineering design process<ol>
<li>Analyze the situation to be emulated, automated or modeled</li>
<li>Based on analysis, construct a skeletal technical system (prototype)</li>
<li>Test the prototype</li>
</ol>
</li>
<li>The process applies &quot;rational engineering principles&quot; only up to the construction of the initial prototype<ul>
<li>The process is no longer an analysis of the initial situation, but the current technical system</li>
<li>Each iteration inserts complexity until, &quot;in the limit, the designed system emulates the target system near-perfectly and does so because it is similarly complex&quot;<blockquote>
<p>Thus one finds that a predictively successful simulation model can be nearly as complex and just as intractable as the b naturalb  system that is modeled by it(see, e.g., Lenhard/Winsberg 2010).This is a technical achievement by rational means. Though it does not consist in the application of b rational engineering principles,b it is not b mereb  tinkering either,but a strategy to systematically optimize the performance of a technical system. </p>
</blockquote>
</li>
</ul>
</li>
<li>Systems Biology vs Synthetic Biology<blockquote>
<p>Systems biology seeks total information and thus incorporates into its representational models all the findings of omics-research, synthetic biology wants to find out how far we can get with what little we know</p>
</blockquote>
</li>
</ul>
</li>
<li>Design vs Evolution...uh oh!<blockquote>
<p>While anti-Darwinian theories of intelligent design are in ill-repute, how can synthetic biology speak of design processes and simultaneously take the insights of evolutionary biology into account?...conceiving the same biological entity at one time as an object of design (as far as that will go), and at another time as an object of evolution (to the extent necessary), synthetic biologists are tending to the boundary between organism and artifact even as they appear to undermine or even reject it.</p>
<ul>
<li>What is the aim of SynBio?  <ul>
<li>Mimic biology? If so, then &quot;artificial selection&quot; is present instead of &quot;natural selection&quot;. i.e. the divices have a purpose, which is not allowed in natural selection and not supported by evolutionary history</li>
<li>Reproduce the work of natural evolution in a purposeful manner? Author asserts that this notion seems troubled by &quot;techonolgical hubris&quot; and accusations of &quot;playing God&quot;, but it can be a sound notion if you treat &quot;nature&quot; as the engineer subject to its own laws.<br>&quot;If there are laws of nature, we can&#39;t violate them anyhow, and in the meantime, it is our job to push the limits of tehcnical possibility&quot;<br>The very question s of how biological engineering should be related to natural history, or of how ev oluti on by natural selection differs from the design of b iological artifacts thus depend on our conception of synthetic biology b  is it an intellectual enterprise with at least some theoretical ambitions or should one judge its attempt to advance unde rstanding of biological systems only in engineering terms ? Only in the former case does the problem arise of having to reconcile the competing ideas of evolution and design , while in the latter case it is merely a practical challenge to insulate as far as possible t he construction of biological entities and processes from the vagaries of evolutionary influence. This fault line between synthetic biolog y as somewhat theory - oriente d and as exclusively engineering - oriented also separates the two ways of paraphras ing Fey nmanb s dictum as well as the two conceptions of rational engineering. It is the fault line that runs between science and technoscience.</li>
</ul>
</li>
<li>I take issue with the statement that if synbio is viewed &quot;only in engineering terms&quot; then one does not have to &quot;reconcile the competing ideas of evolution and design&quot;. What if part of the engineering process requires knowing how the original system came to be in order to further modify it, as in complex system design? <ul>
<li>I think it&#39;s a copout to assume that engineers can assume away issues that may be relevent to design because they don&#39;t have &quot;theoretical ambitions&quot;</li>
<li>What happens when design (synbio) meets evolution (wild-type organisms). The author asserts this consideration should enter at the beginning of design &quot;in reflections on the rhyme and reason of naturally evolved biological complexity&quot;. Doesn&#39;t &quot;rhyme and reason&quot; imply a purpose</li>
</ul>
</li>
</ul>
</blockquote>
</li>
<li>Why study the origin of synbio&#39;s?<ul>
<li>Normative implications of historical accounts, &quot;suggesting what synthetic biology ought to be and what opportunities and risks it poses, what obligations and expectations come with it&quot;</li>
<li>Is synbio an intellectual paradigm-shift from systems biology that revolves around systems-thinking to understand complexity (synbio is applied systems bio) or is it a means of generating higher complexity by treating systems as more or less efficient units of technical functioning?</li>
</ul>
</li>
<li>Synbio is the application of systems bio<ul>
<li>Normative behavior = synbio  must properly apply systems bio</li>
<li>&quot;Systems&quot; are dynamic structures that require general systems theory or a theory of non-linear complex dynamics to describe it</li>
<li><em>Irreducible Holism</em> triumphs over <em>Reductionist Mechanism</em></li>
</ul>
</li>
<li>Synbio is a Technical Opportunity<ul>
<li>&quot;Systems&quot; are a technical construct that consists of at least sever interacting parts<ul>
<li>NOT a holistic way of thinking</li>
</ul>
</li>
<li>Normative behavior = what amount of complexity can I generate with the least amount of theoretical understanding?<ul>
<li>iGEM illustrates this example (even undergraduates can &quot;do&quot; synthetic biology!!)</li>
<li><em>Reductionist Mechanism</em> does not necessarily triumph over <em>Irreducible Holism</em>, the questions are neither answered nor dismissed. Rather, they are absorbed into an &quot;engineering idiom&quot;</li>
</ul>
</li>
</ul>
</li>
<li><strong>SUMMATION OF SYSTEMS THINKING VS TECH OPPORTUNITY (scientific vs technoscientific)</strong><blockquote>
<p>To put it a bit metaphorically, then, in relation to systems biology , the two stories about synthetic biology do not attribute to the researchers different theories, opinions , or beliefs but an entirely different mind - set , a different way of living the laboratory life, of participating in history and relating to the tradition of science and the Enlightenment. On the one hand, t here is the scientific mind - set of those who query the limits of reductionism and embrace systems thinking ; on the other hand is the technoscientific mind - set of those who no longer seek the most appropriate way of reducing complexity and promoting intellectual understanding , but who proceed instead to generate biological complexity from available theories and techniques.</p>
</blockquote>
</li>
<li>Conflict<ul>
<li>Researchers may work using both methods, but within publications they only demonstrate one way of thinking</li>
<li>If Synbio tries to have it all (basic and applied science) then there are three conflicts that emerge <ol>
<li>Accommodating ignorance</li>
<li>Discontinuous continuities</li>
<li>The matter of definition</li>
</ol>
</li>
<li>Accommodating ignorance<ul>
<li>What happens when senior researchers confront iGEM teams that &quot;impatiently seek to achieve on extremely short time scales what others frame in terms of multidisciplinary, sometimes career-long research trajectories&quot;?</li>
<li>Five hard truths for synbio - Roberta Kwok<ul>
<li>Problem: No standardization </li>
</ul>
</li>
<li>Answer: sub relative for absolute measures<ul>
<li>Problem: No predictable design proceedures and trial and error is too arduous<ul>
<li>Answer: Directed evolution, i.e. the design cycle and its  iteration &quot;until the system is optimized&quot;</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li>Problem: Greater system complexity<ul>
<li>Answer: Automation</li>
</ul>
</li>
<li>Problem: Unexpected interactions<ul>
<li>Answer: Insulate biological machinery from cell&#39;s &quot;natural machinery&quot;</li>
</ul>
</li>
<li>Problem: Variability and stability<ul>
<li>Answer: Avoid variability and increase stability</li>
</ul>
</li>
<li><strong>Summary</strong>: Only chance for synbio to succeed is by way of design processes that can acoommodate or work around ignorance<ul>
<li>Nature published an editorial on the 10th anniversary of the repressilator and a switchable regulartory network and say that these &quot;contributions to and from basic science are part of synthetic biology that most deserves celebration&quot;<ul>
<li>Does this mean synbio is a basic science? an applied science? if applied science then:</li>
</ul>
</li>
<li>&quot;in which sense...is the design of the repressilator an &#39;experiment&#39;&quot;?</li>
<li>Is &quot;to understand something one must build it&quot; really a principle of physics as a basic science?</li>
<li>How and when is &quot;to design&quot; the same thing as &quot;to define&quot;?</li>
<li>How is the ability to create a computer model the same as human knowledge, ex. theories that are tractable by the human mind<ul>
<li>Remember that the quote is &quot;What I cannot create, I do not understand&quot; not &quot;What I can create, I do understand&quot;, which implies that &quot;understanding&quot; can leap ahead of explanation and prediction</li>
</ul>
</li>
<li>Discontinuous continuities<br><img src="/papers/Nordmann2014_F1.png" alt="Nordmann Figure 1"><ul>
<li>From Bujara and Panke, 2010); Issues;</li>
</ul>
</li>
<li>The rest of the paper was highlighting discontinuous differences between design by Random Evolution, Directed Evolution and Comginatorial Design</li>
<li>Design by Random Evolution (evolutionary design) is an oxymoronic misnomer...see intelligent design</li>
<li>The picture implies further development of combinatorial design strategies will produce knowledge required for progression to rational design</li>
<li>Rational design is assumed to be more effective than evolutionary design...why is that? because it&#39;s &quot;rational&quot;, or knowledge based? The authors admit that it&#39;s just a hypothesis that needs to be tested.</li>
<li>The figure paints an ambivalent picture: the knowledge required is too great so a new trajectory is needed that bypasses the prohibitively high knowledge requirement, or the picture cheerfully tells a story of progress and a sequence of steps towards the ultimate goal of rational design</li>
</ul>
</li>
<li>Tabor advocates creating things through a kind of technical probing and then passing the information on to systems biologists for hypothesis generation and testing<blockquote>
<p> b As automated circuit design and assembly dovetails with iterative optimization, our ability to engineer circuits should extend beyond our ability to truly understand how they work. The tractability of modularly constructed synthetic circuits, however, should also feed back to accelerate the cycle of hypothesis generation and testing in systems biology.b  (Tabor 2012, 1063)</p>
<ul>
<li>Tabor&#39;s process:<br>Tabor offers an epistemologically telling description of the design - cycle approach: b Here, the first design is based on the ligand - inhibited repressors LacI and TetR. Each is initially placed upstream of an associated fluorescent reporter on a polycistronic mRNA. The operons show poor reporter expression, which is then imp roved by b plugging inb  additional copies of the appropriate promoter upstream of each reporter. This increases reporter expression, but reveals that the circuit cannot reach the TetR - dominated state. The tetR promoter is then swapped for a stronger version , but this overcompensates for the problem making only the TetR state stable. A library of random tetR ribosome binding sites (RBSs) is then screened, and a variant that hits the bistable sweet spot is found b  (Tabor 2012, 1063, compare Litcofsky et al. 2012)</li>
</ul>
</blockquote>
</li>
<li>The author takes Tabor as saying that computers, rather than humans, will &quot;understand&quot; the organizing principles of life, which the author argues &quot;leaves untouched the break with the epistemic values and traditional ideals of science&quot;<ul>
<li>The matter of definition</li>
</ul>
</li>
<li>Synthetic biology is not analytic because it generates rather than reduces complexity</li>
<li>The author&#39;s definition of synbio:<blockquote>
<p>This, then, suggests a definition of synthetic biology that highlights its specific epistemic values and ideals: For constructive purposes synthetic biology builds on the achievement in silico, in vitro, and in vivo of technical control of biological complexity, that is, it is the endeavor of drawing together de facto achievements of technical control for the generation of technical systems with greater biological complexity.</p>
</blockquote>
</li>
<li>Conclusion</li>
<li>The tension between an engineering approach and the quest for understanding biological processes cannot be resolved by simply stating that synbio can do both<blockquote>
<p>&quot;It is quite impossible to simply marry the epistemic ideals of technoscientific synthetic biology to those of biology as a theoretical science - they pull in opposite directions, after all: Here the reduction of complexity for the purposes of intellectual tractability, there the drawing together of scientific knowledge and technological capability for the generation of complexity beyond our ability to truly understand how our own creations work. Here the identification of bottle-necks and needs- to-know for rational engineering, along with the demand for more and better theoretical knowledge in order to diminish ignorance, and there the attempt primarily to discover how much one can achieve even with how little we know, with considerable tolerance for ignorance of everything that can be black-boxed. The tension, even antagonism, between these epistemic ideals cannot be dissolved b  which does not preclude, of course, that the corresponding research findings can inform, even inspire one another.</p>
</blockquote>
</li>
<li>YES!<blockquote>
<p>It is not at all impossible, in contrast, to marry the notion of bringing an engineering approach to biology and the notion of knowledge production through synthetic biology b  but in order to do so one might have to become dishabituated from the established scientific image of knowledge and of knowledge production. From the point of view of the engineering approach, knowledge and understanding need not be tied to the intellectual tractability of causal relations, nor does it consist in the truth or falsity, or empirical adequacy of linguistic statements such as theories or hypotheses. Instead, knowledge and understanding might reside in computer models and other technologically robust constructions, tied to the iterations of the design cycle as a learning process of sorts.</p>
</blockquote>
</li>
<li>YES YES! From the Nature editorial:<blockquote>
<p>As it develops along this and other paths, synthetic biology itself will demand more by way of new fundamental biological knowledge b  quantitative, systematic, computational and biophysical. And conversely, one of the deepest lessons from these first ten years is that biological knowledge will require synthetic approaches if it is to become a mature and reasonably predictive science. (Nature editorial 2010)</p>
</blockquote>
</li>
</ul>
</li>
</ul>
</li>
<li><p><a href="/papers/Bujara2010.pdf">Engineering in complex systems</a></p>
<ul>
<li>Measure, Model, Manipulate?<blockquote>
<p>The hallmark of design is the engineering design cycle with its three elements of &#39;measure, model, manipualte&#39;</p>
<ul>
<li>This does not resemble any engineering design process or cycle that I am familiar with and it was published in &quot;current opinions in Biotechnology&quot;!!! :O</li>
<li>The &#39;measure, model, manipulate&#39; cycle appears to be an assumption the rest of the paper is based upon</li>
</ul>
</blockquote>
</li>
<li>Measure<ul>
<li>Problem: Limited observation times to prevent cellular adaptation</li>
<li>Possible solution: Doug&#39;s assay &quot;memory&quot; device??</li>
</ul>
</li>
<li>Limitations of cycle<ul>
<li>Two methods to reduce complexity<ol>
<li>Reduce system size</li>
<li>Add insulation between subsystems</li>
</ol>
</li>
</ul>
</li>
</ul>
</li>
<li><p><a href="/papers/Agapakis2013.pdf">Designing Synthetic Biology</a></p>
</li>
<li><p><a href="/papers/nestreport2005.pdf">Synthetic Biology: Applying Engineering to Biology</a></p>
<ul>
<li><p>Never defines what engineering is</p>
<blockquote>
<p>Adapting natural biological mechanisms to the requirements of an engineering approach, the possibilities for re-assembling biological systems in a designed way will increase tremendously</p>
</blockquote>
<ul>
<li>What are those requirements?</li>
</ul>
</li>
<li><p>Confuses managing complexity with the engineering method</p>
<blockquote>
<p>the introduction of design principles such as modularity of parts, standardization of parts and devices according to internationally recognized criteria, and the (reciprocal) adaptation of available abstract design procedures to biological systems, coupled to novel technological breakthroughs (such as cheap mass synthesis of large DNA segments) that allow the decoupling of design and fabrication, will fundamentally change our current concepts of how to manipulate biological systems.</p>
</blockquote>
</li>
<li><p>What is this &quot;engineering approach&quot; you speak of?</p>
</li>
</ul>
</li>
</ul>
<blockquote>
<p>Synthetic biology is not content with explaining or simply reproducing the behaviour of natural systems. Rather, synthetic biology aims to go one step further by building, i.e. synthesizing, novel biological systems from scratch using the design principles observed in nature but with expanded, enhanced and controllable properties. The complexity of such a design goal makes an engineering approach imperative</p>
</blockquote>
<ul>
<li>Great bibliography! Papers listed by category</li>
</ul>
<ul>
<li><p><a href="/papers/OMalley2011.pdf">Exploration, iterativity and kludging in synthetic biology</a></p>
<ul>
<li>Three &quot;streams&quot; in synbio that &quot;produce knowledge&quot; in different ways<ol>
<li>DNA-based device construction</li>
<li>Genome-driven cell engineering</li>
<li>Create protocells</li>
</ol>
</li>
<li>Create Protocells<ul>
<li>Make minimal or minimized cells that can function as basic approximations of living cells </li>
<li><em>Possibly remove complexities from the evolutionary process, much in the same way you could make x86 more efficient by removing vestiges from its past?</em></li>
</ul>
</li>
<li>&quot;In order to achieve the goal of constructing designed and decomplexified systems, synthetic biologists...argue engineering approaches will work only if the excess or irrational complexity of living systems is minimized or, better, avoided altogether&quot;<ul>
<li><em>The authors seem to want just the benefits of the engineering process, without the actual process</em> </li>
<li><blockquote>
<p>Rather than hypothesis testing in a formal and narrow sense, synthetic biology invokes a range of philosophically neglected strategies for gaining scientific knowledge.</p>
<ul>
<li><em>why the assumption that synthetic biologists want to gain scientific knowledge?</em></li>
</ul>
</blockquote>
</li>
</ul>
</li>
</ul>
</li>
<li><p><a href="/papers/andrianantoandro2006.pdf">Synthetic biology: new engineering rules for an emerging discipline</a></p>
</li>
</ul>
<h1>Real Time Reconfiguration</h1>
<ul>
<li><a href="/papers/mccaskill2010.pdf">FPGA Partial Reconfiguration Goes Mainstream</a><ul>
<li>Article starts on pg 42</li>
</ul>
</li>
<li><a href="/papers/prtutorial.pdf">Partial Reconfiguration: A Simple Tutorial</a></li>
<li><a href="/papers/pr_tutorial.pdf">Xilinx UG947</a> </li>
</ul>
<h1>FPGA Lab Automation</h1>
<ul>
<li><a href="/papers/wang2012.pdf">FPGA-based bio-cybernetic system for lab-on-a-chip automation</a></li>
</ul>
<h1>Papers for RPC</h1>
<ul>
<li><a href="/papers/uliana2014.pdf">FPGA-Based Accelerator Development for Non-Engineers</a><ul>
<li>It used to be that data production was the bottleneck, now the bottleneck is data consumption and &quot;intelligent processing at reasonable rates&quot;</li>
<li>They use the National Instruments LabVIEW FPGA environment targeting a Convey HC-1 platform</li>
<li>&quot;FPGAs outperform CPUs and GPUs for many applications, due to support for non-standard data types and massive, fine-grain parallelism, features that dominate many bioinformatics applications&quot;</li>
<li>Reprogrammability is a feature inherent to FPGAs and is not available in ASICs.</li>
<li><strong>Meh, not a good paper for RPC</strong></li>
</ul>
</li>
<li><a href="/papers/homsirikamol2014.pdf">Can High-Level Synthesis Compete Against a Hand-Written Code in the Cryptographic Domain? A Case Study</a></li>
<li><a href="/papers/drahonovsky2014.pdf">A Highly Flexible Reconfigurable System on a Xilinx FPGA</a></li>
<li><a href="/papers/johnson2014.pdf">Fault Attack on AES via Hardware Trojan Insertion by Dynamic Partial Reconfiguration of FPGA over Ethernet</a></li>
<li><a href="/papers/granado2009.pdf">A new methodology to implement the AES algorithm using partial and dynamic reconfiguration</a></li>
<li><a href="/papers/preuber2014.pdf">PoC-Align: An Open-Source Alignment Accelerator using FPGAs</a></li>
<li><a href="/papers/luo2014.pdf">Side-challen Power Analysis of Different Protection Schemes Against Fault Attacks on AES</a></li>
<li><a href="/papers/aasaraai2014.pdf">What Limits the Operating Frequency of a Soft Processor Design</a></li>
</ul>

        </div>
    </body>
</html>
