<!DOCTYPE html>
<html lang="en">
    <head>
	    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <title>Do Work!</title>
        <meta name="description" content="Ryan Silva's Lab Work">
        <meta name="author" content="Ryan Silva, Branched from Todd Branchflower">
        <link rel='stylesheet' type='text/css' href='//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.min.css'>
        <script src='//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/js/bootstrap.min.js'></script>
	<script type="text/x-mathjax-config">
		  MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'],
		                  ['\\(','\\)']]}});
	 </script>
	<script type="text/javascript"
		   src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
         </script>
	<link rel="stylesheet" href="http://yandex.st/highlightjs/8.0/styles/default.min.css">
	<script src="/node_modules/highlight.js/lib/index.js"></script>
    </head>
    <body>
        <div class = "container">
            <div class = "navbar navbar-inverse">
                <div class = "navbar-inner">
                    <a class = "brand" href="/index.html">AVLIS</a>
                    <ul class = "nav">
                        <li><a href="/">Home</a></li>
                        <li><a href="/setup/index.html">Setup</a></li>
                        <li><a href="/equipment/index.html">Equipment List</a></li>
                        <li><a href="/lab/index.html">Lab Work</a></li>
                        <li><a href="/links/index.html">Links</a></li>
                        <li><a href="/papers/index.html">Papers</a></li>
                        <li><a href="/TeamDocs/index.html">Team Docs</a></li>
                        <li><a href="/action_items/index.html">Action Items</a></li>
                    </ul>
                </div>
            </div>
            <h4><a href="/papers/pH_based_biosensor_arsenic_drinkingwater.pdf">pH-based biosensor for detection of arsenic in drinking water</a></h4>
<p><img src="pHResponse_pH6to8.png" alt="pH Response"></p>
<h4><a href="/papers/Edinburgh_Development_arsenic_drinking_water.pdf">Development of a nir fovel biosensor for the detection of arsenic in drinking water</a></h4>
<ul>
<li>World Health Organization recommended limit is 10 ppb or <a href="http://www.who.int/mediacentre/factsheets/fs372/en/">10 $\mu$g/litre</a></li>
<li>Sensor can detect 5 ppb, albeit with a caveat<ul>
<li>You must wait 500 minutes before the Output Dynamic Range between 0ppb and 5ppb goes above .1pH<h5>pH vs Time:</h5>
</li>
</ul>
</li>
</ul>
<p><img src="pHresponse.png" alt="pH Response"></p>
<h4><a href="/papers/NanowireFET_Nanotechnology.pdf">Optimization of pH sensing using silicon nanowire field effect transistors with HfO2 as the sensing surface</a></h4>
<p><img src="IV.png" alt="IV Curve"></p>
<h4><a href="/papers/nrm3767.pdf">Realizing the potential of synthetic biology</a></h4>
<h4><a href="/papers/Engineering_EColi_tosee_light.pdf">Engineering E Coli to see light</a></h4>
<h4><a href="/papers/Spatiotemporal.pdf">Spatiotemporal control of cell signalling using a light-switchable protein interaction</a></h4>
<h4><a href="/papers/Synthetic_edge_Detection.pdf">A Synthetic Genetic Edge Detection Program</a></h4>
<h4><a href="/papers/Microfluidics_and_SynBio.pdf">Combining microfluidics and synthetic biology</a></h4>
<h4><a href="/papers/uBioRobots.pdf">Sensors for Micro Bio Robots via Synthetic Biology</a></h4>
<h4><a href="/papers/thorsen2002.pdf">Microfluidic Large-Scale Integration</a></h4>
<ul>
<li>Review paper for the state of the microfluidic art in 2002<blockquote>
<p>A key component of these networks is the fluidic multiplexor, which is a combinatorial array of binary valve patterns that exponentially increases the processing power of a network by allowing complex fluid manipulations with a minimal number of inputs.</p>
</blockquote>
</li>
</ul>
<p><img src="mux.png" alt="Microfluidic Multiplexor!"></p>
<blockquote>
<p>On the basis of the utility of these examples, we believe that other concepts developed for electronic integrated circuits can be usefully transferred to chemical and biochemical analysis and processing in microfluidic devices</p>
</blockquote>
<h4><a href="/papers/Mammalian_pHSensor.pdf">A Synthetic Multifunctional Mammalian pH Sensor and CO2 Transgene-Control  Device</a></h4>
<ul>
<li>Device pitched as a treatment for diabetes<blockquote>
<p>Life operates...within a narrow biochemically permissive range (e.g., pH<br>7.35-7.45 in humans</p>
</blockquote>
</li>
</ul>
<h4><a href="/papers/Abstraction_Layers.pdf">Abstraction Layers for Scalable Microfluidic Biocomputers</a></h4>
<h4><a href="/papers/Tuning.pdf">Tuning Response Curves for Synthetic Biology</a></h4>
<blockquote>
<p>Synthetic biology includes a concerted effort to formalize an engineering discipline suitable for the design and implementation of novel biological systems.</p>
</blockquote>
<ul>
<li>Biology operates in a noisy environment</li>
<li>Response curve defined by a single first order differential equation, which is a highly approximate model based on &quot;empirical observations&quot; </li>
</ul>
<h5>Hill Function</h5>
<p>$$\theta(x)=\frac{x^n}{K^n+x^n}$$</p>
<ul>
<li>$x$ = unbound ligand concentration</li>
<li>$\theta(x)$ = avg fraction of binding sites occupied by an input ligand as a function of unbound ligand concentration.</li>
<li>$K$ = The Hill constant. Rough indicator of the level of ligand concentrations needed to induce saturation ($x \gg K$)</li>
<li>$n$ = The Hill coefficient. <strong>The larger the value of $n$, the steeper the slope of the Hill function</strong><ul>
<li>$n &gt; 1$ indicates cooperativity, where affinity increases in the presence of previously bound ligands</li>
<li>$0 &lt; n &lt; 1$ indicates negative cooperativity, where affinity is reduced</li>
<li>$n = 1$ indicates a noncooperative reaction<br><img src="Tuning_Fig2.png" alt="Figure 2"></li>
</ul>
</li>
</ul>
<h5>Equation for Protein Expression</h5>
<p>$$\frac{dy}{dt}=k&#39; + k \left(\frac{x^n}{K^n+x^n} \right)$$</p>
<ul>
<li>$y$ is the concentration of the protein being expressed</li>
<li>$k&#39;$ is the basal rate of production</li>
<li>$k$ is the maximum additional production rate arising from up-regulation</li>
<li>$\left(\frac{x^n}{K^n+x^n} \right)$ is the increasing sigmodal Hill function</li>
</ul>
<p>Equation for Protein Repression</p>
<p>$$\frac{dy}{dt}=k&#39;+k\left(\frac{K^n}{K^n+x^n}\right)$$</p>
<ul>
<li>$k&#39;+k$ is the basal expression rate<ul>
<li>$k&#39;$ accounts for the fact that complete repression may not be possible</li>
</ul>
</li>
<li>$\left(\frac{K^n}{K^n+x^n}\right)$ is the decreasing sigmodal Hill function</li>
</ul>
<h5>Types of Tuning</h5>
<p><img src="tuning_fig1.png" alt="Types of Tuning"></p>
<ul>
<li><strong>Vertical Scaling</strong> <ul>
<li>Change the promoter-gene copy number<ul>
<li>Multiplies the rate at which mRNA is produced and therefore translated</li>
</ul>
</li>
<li>Change RBS strength</li>
<li>Codon optimization</li>
</ul>
</li>
<li><strong>Vertical Shifting</strong><ul>
<li>Introduce or tune a constitutive (always on) source of output $y$</li>
</ul>
</li>
<li><strong>Vertical Extension</strong><ul>
<li>Up-Regulation: Tune the activation potency of bound TF protein </li>
<li>Down-Regulation: No known single-step method (excluding $k&#39;=0$ case, which is identical to vertical scaling)</li>
</ul>
</li>
<li><strong>Leakage</strong><ul>
<li>Up-Regulating: Leakage is that which is always active<ul>
<li>No known direct method</li>
</ul>
</li>
<li>Down-Regulating: Leakage is that which cannot be repressed<ul>
<li>Vary the repression strength of each bound TF protein </li>
</ul>
</li>
</ul>
</li>
<li><strong>Horizontal Scaling</strong><ul>
<li>Tune the Hill constant $K$ (increasing $K$ scales curve to the right)<ul>
<li>$K$ is related to the effective binding affinity of the input signal to the process</li>
</ul>
</li>
<li>Tune the binding affinity of the TF to the promoter</li>
</ul>
</li>
<li><strong>Steepness</strong><ul>
<li>Tune the Hill coefficient $n$ (increasing $n$ leads to increasing steepness)</li>
<li><em>Ultrasensitivity:</em> A biochemical process that has a steep or switch-like steady-state response curve</li>
<li>Requires adjusting the effective binding cooperativity<ul>
<li>Implies the cooperative binding of multiple TFs to the same promoter</li>
</ul>
</li>
</ul>
</li>
<li><strong>Dynamic Range</strong><ul>
<li>The observable space between uninduced and fully saturated induction</li>
</ul>
</li>
</ul>
<h3><a href="/papers/soft_litho_CAD.pdf">CAD for Microfluidic Chips Based on Multilayer Soft Lithography</a></h3>
<blockquote>
<p>A second difference between electronic CAD and microfulidic CAD is the fast turn-around time in manufacturing microfluidic chips.</p>
</blockquote>
<p>I argue that prototyping using FPGAs is significantly faster than manufacturing microfluidic chips.</p>
<h3><a href="/papers/Recent_uF_Developments.pdf">Recent developments in microfluidic large scale integration</a></h3>
<ul>
<li>A programmable uF platform has kind-of been investigated, albeit just for mixing and processing via fluorescent derivatization.<ul>
<li><a href="/papers/Programmable_uF_automaton.pdf">Digitally programmable microfluidic automaton for multiscale combinatorial mixing and sample processing</a><ul>
<li>The review paper quotes the shortcomings of the device as: <blockquote>
<p>larger hybrid PDMS/glass valve technology differentiates itself from mLSI and is more suitable for microliter scale fluid volue processing (the chamber volume is 120 nL); moreover this technology requires one external valve per chamber and it seems unlikely that any reduction is possible</p>
</blockquote>
</li>
</ul>
</li>
</ul>
</li>
<li>On second glance, an FPGA-like uF platform has been done...damn:<ul>
<li><a href="/papers/FPGA_uF.pdf">A software-programmable microfluidic device for automated biology</a></li>
</ul>
</li>
<li>And again...This time using a combination of flow based and electrowetting-enabled droplet based uF<ul>
<li><a href="/papers/Programmable_electrowetting_uF.pdf">A programmable droplet-based microfluidic device applied to a multiparameter analysis of single microbes and microbial communities</a><blockquote>
<p>This is largely due to the prevailing para-digm in microfluidic research in which devices areb hardwiredb  for specific fluid handling tasks, necessitating a customized design for each application or change in protocol. This application-spe-cific approach requires iterative cycles of device design, fabrica-tion, and testing, presenting a major obstacle to the development of new applications and limiting user adoption and community access. In analogy to how programmable integrated circuits en-abled a broader community of developers and nonexpert users,the advancement of programmable microfluidic devices standsto dramatically enhance the pervasiveness and impact of micro-fluidic systems (15)</p>
</blockquote>
</li>
</ul>
</li>
<li><strong>Idea</strong>: what if you combine MHDL with an actual chip by building a Place-and-Router?<ul>
<li>or what if you built a programmable mLSI chip that could be addressed by MHDL, or a subset thereof?</li>
</ul>
</li>
<li><strong>This doesn&#39;t seem to have been investigated!</strong><ul>
<li>The electrowetting device uses a LABVIEW program to control the chip<ul>
<li><a href="Methods_ProgramElectrouF.pdf">Methods used to make device</a></li>
</ul>
</li>
<li>The claimed FPGA-like devices uses Visual Basic to address the solenoid elements</li>
</ul>
</li>
<li>The evolution of the FPGA began with the HDL, not the other way around...right?<h3>Microfluidic Hardware Description Language (MHDL)</h3>
</li>
<li>MHDL is a specification language for mLSI chip architectures, as opposed to assay specification. An assay specification language compatible with MHDL is<br>Biocoder as shown in this paper: <a href="/papers/MHDL_Synth.pdf">Automatic Synthesis of Microfluidic Large Scale Integration Chips from a Domain-Specific Language</a></li>
<li>Introduced here: <a href="/papers/MHDL.pdf">Design and Verification Tools for Continuous Fluid Flow-based Microfluidic Devices</a></li>
</ul>
<h1>Characterization</h1>
<h3><a href="/papers/Tabor_OptoChar.pdf">Optogenetic characterization methods overcome key challenges in synthetic and systems biology</a></h3>
<blockquote>
<p>Component signal processing properties, such as the relative amplitude (gain) and time delay of output relative to input for various signals, are captured using <strong>phenomenological</strong>, or &#39;black box&#39; mathematical models. The models are used to reliably assemble components into circuits with more advanced capabilities, such as band-pass filters or amplifiers, and eventually into complete systems, such as computers or radios</p>
</blockquote>
<ul>
<li>i.e. Derive models from experimentation, NOT FROM BIOLOGICAL THEORY!</li>
</ul>
<blockquote>
<p>Given sufficient measurements, one should be able to formulate a mathematical model that accurately predicts receptor outputs for input signals that have not been tested. Because the only purpose of the model is to predict receptor I/O, a black-box model will typically be preferred over a more complicated version reflecting the receptor molecular biology. Once it is parameterized with experimental data, the model can be used to simulate the receptor response to a wide range of different inputs in silico, thus enabling the computational design of effector signals capable of driving biological signals with desired quantitative and temporal features and satisfying challenge two. Additionally, if receptor outputs can be measured using fluorescent proteins in real time, an in silico feedback controller can adjust the designed input signal to account for any deviations from the model prediction</p>
</blockquote>
<h2><a href="/papers/Evolution.pdf">Evolution Paper</a></h2>
<h2><a href="/papers/ApplicationsSynBio.pdf">Synthetic Biology and the Art of BioSensor Design</a></h2>
<h2><a href="http://pubs.acs.org/doi/full/10.1021/ac9805201">Biosensor for Direct Determination of Organophosphate Nerve Agents using Recombinant E.Coli</a></h2>
<p><img src="/papers/experimental_setup.gif" alt="Experimental Setup"></p>
<h2><a href="/papers/Analog_Synbio.pdf">Analog Synthetic Biology</a></h2>
<blockquote>
<p>Nevertheless, we must not forget that digital computation has not offered an effective paradigm for computing efficiently and precisely with noisy and unreliable devices; that multi-logic-gate computations can impose significant metabolic or toxicity burdens on cells owing to their need to use a lot of parts and power; that the fact that there are five to six orders of magnitude fewer genes per cell than digital transistors per chip means that using genes to only perform logic is likely not an efficient way to attain high complexity; that a library of b digital partsb  with good onb off ratios, low standby power consumption and low crosstalk does not exist in biology; that the computing basis functions in cells are not really logic functions and abstracting them as such compromises computational efficiency; and that logic basis functions are not the only universal computation primitives.</p>
</blockquote>
<ul>
<li>No static discipline? Don&#39;t just make one up that doesn&#39;t work the same way in cells as it does in computers!</li>
</ul>
<blockquote>
<p>The b 1b s and b 0b s of todayb s digital computers are useful abstractions of the analog signals in cells, but are often an oversimplification.</p>
</blockquote>
<ul>
<li><p>If digital abstraction can&#39;t be used to create a working system, then it isn&#39;t digital abstraction!</p>
<blockquote>
<p>Furthermore, as in analog circuits, the wiring of the output of one circuit to the input of another leads to b loadingb  interactions that degrade overall function and prevents simple modular digital abstractions from being effective [13].</p>
</blockquote>
</li>
<li><p>Figure 1b allows you to focus on a <code>protein input protein output</code>point of view<br><img src="/papers/analog.png" alt="analog"></p>
</li>
</ul>
<h2><a href="/papers/voigtNORGate.pdf">Voigt Nor Gate</a></h2>
<h1>Cell to Cell communication applications</h1>
<ul>
<li><a href="/papers/felix2013.pdf"> Spatial Structure Facilitates Cooperation in a Social Dilemma: Empirical Evidence from a Bacterial Community</a></li>
<li><p>GASP = Growth Advantage in Stationary Phase</p>
<blockquote>
<p>The fact that cooperation is simultaneously ubiquitous and fragile, begs for an explanation regarding its emergence and maintenance.<br>E. coli bacteria collectively restrain growth to avoid population collapse when resources become scarce [21]b [23]. The collective decision to restrain growth before all resources are depleted is a form of altruistic cooperation, and one can think of the entry into stationary phase as a social contract: a collective rule of density-dependent growth regulation [9], [24]. The long-term benefits of going into stationary phase are evident: by limiting population growth, bacteria ensure that enough resources will be available to maintain a viable population for an extended period, and survive until conditions improve and growth can be resumed. In the short term however, b cheaterb  cells that ignore the collective decision to restrain growth and that utilize the scarce resources to proliferate instead of investing in maintenance, may increase in numbers. As a result, cheaters jeopardize the survival of the collective b  which in turn could face a tragedy of the commons [25].<br>In a Prisoner&#39;s Dilemma, the Sucker&#39;s payout is negative for their model (only look at the sign of the payoff differences) to work; in a Snowdrift regime the Sucker&#39;s payout is positive). </p>
</blockquote>
</li>
<li><p><a href="/papers/Molina2014.pdf">Cooperation and bacterial pathogenicity: an approach to social evolution</a></p>
</li>
<li><p><a href="/papers/Goers2014.pdf">Co-culture systems and technologies: taking synthetic biology to the next level</a></p>
<ul>
<li>Review paper on experimental and theoretical co-culture set-ups in nature and synbio</li>
<li>Co-culture setups exist to study cell-cell interactions of any kind</li>
<li>Current co-culture systems (methods of separating cell populations<br><img src="Goers2014_F3.jpg" alt="Drawing" style="width: 500px;"/></li>
<li><strong>Does TweeColi offer a novel method of conducting co-culture experiments?</strong><blockquote>
<p>As population interactions suffer from context dependency and can change in different environments, characterization should include stress limits. For example, an engineered interaction was reported to be functional only within certain pH ranges [39]. Taking context dependency into account does not imply a departure from the engineering and computing analogy on which synthetic biology is based, but rather an improved appreciation of the challenges that come with the analogy. Just as civil engineers need to consider the environmental stresses that a building will have to withstand, synthetic biologists need to consider the working environment of bio-parts at the stage of circuit design. On the other hand, co-culture technologies can support the genetic engineering that is at the basis of synthetic systems. Genetic programming cannot entirely predict cellular behaviour, as environment plays a role and there are still limits in what can be controlled genetically [41]. Using genetic engineering and controlled environmental conditions through experimental set-up together could allow a level of population control that was previously thought impossible. Emergent and unpredicted behaviour can arise in co-cultures and may or may not be desired. One important question in the choice of cultivation technology is what properties of the natural populations are to be preserved or suppressed. Overall, populations can be controlled genetically and through the environment. Which aspects can be controlled genetically or through set-up is a subject of ongoing research.</p>
</blockquote>
</li>
</ul>
</li>
<li><a href="/papers/Bacchus2012.pdf">Engineering of synthetic intercellular communication systems</a><ul>
<li>Review paper of intercellular communication and multicellular consortia</li>
<li>Can we do the population control experiment in A below using TweeColi?<br><img src="/papers/Bacchus2012_F1.jpg" alt="Bacchus figure 1"></li>
</ul>
</li>
<li><a href="http://link.springer.com/article/10.1007/s00792-009-0261-3/fulltext.html#CR84">Cultivating the uncultured: limits, advances and future challenges</a> &lt;- No pdf link<ul>
<li>Culture isolation using pH control?<br><img src="culture_isolation.gif" alt="culture_isolation.gif"></li>
</ul>
</li>
</ul>
<h1>Synthetic Biology Design</h1>
<ul>
<li><a href="/papers/Lewens2013.pdf">From <em>bricolage</em> to BioBricks: Synthetic Biology and rational design</a><ul>
<li>Explores the limitations of the synbio design process </li>
<li>Synthetic Biology defined by the European Commission as &quot;the synthesis of complex, biologically based (or inspired) systems which display functions that do not exist in nature. This engineering perspective may be applied at all levels of the hierarchy of biological structures...In essence, synthetic biology will enable the design of &#39;biological systems&#39; in a rational and systematic way&#39; <blockquote>
<p>The UK Health and Safety executive says that synthetic biology can be described as b the design and construction of new biological parts, devices and systems, and the redesign of existing, natural biological systems for useful purposesb  (HSE, 2007). The European Commission says that synthetic biology is b the engineering of biology: the synthesis of complex, biologically based (or inspired) systems which display functions that do not exist in nature. This engineering perspective may be applied at all levels of the hierarchy of biological structures b &amp; In essence, synthetic biology will enable the design of b biological systemsb  in a rational and systematic wayb  (European Commission, 2005). The UK Parliamentary Office of Science and Technology tells us that synthetic biology b describes research that combines biology with the principles of engineering to design and build standardised, interchangeable biological DNA building-blocks. These have specific functions and can be joined to create engineered biological parts, systems and, potentially, organisms. [Synthetic biology] may also involve modifying naturally occurring genomes b &amp; to make new systems or by using them in new contextsb  (POST, 2008).</p>
</blockquote>
</li>
<li>&#39;Bottom up&#39; design = construction of genetically specified components and combine them to create novel organic devices</li>
<li>Top down = begin with naturally occurring organisms and seek to remove redundant or unnecessary parts to produce simpler, less &#39;noisy&#39; organic machines.</li>
<li>Research Opportunity?<blockquote>
<p>the notion that an engineering design perspective characterises a variety of synthetic biology approaches remains in very recent research articles</p>
</blockquote>
</li>
<li>Irrational vs Rational design process<blockquote>
<p>It is nonetheless reasonable to place synthetic biology towards one end of a continuum between purely b blindb  processes of organic modification at one extreme, and wholly rational, design-led processes at the other. </p>
<ul>
<li>Irrational:<br>They are b irrationalb  only because the agent who oversees the process need not understand why the resultant entity works as it does. Whether a method is b rationalb  or b irrationalb  is itself something that comes by degrees: an engineer may not understand why the parts of a machine work as they do, but he may still be able to predict in rough-and-ready ways how they will behave when brought together. After several moderately successful attempts he may be surprised at the makeup of the machine that works best.</li>
</ul>
</blockquote>
</li>
<li>Nazi attempt to breed back aurochs failed because<blockquote>
<p>On the basis of vague criteria and without proper knowledge of the appearance of the aurochs, the two brothers made inaccurate selections among the crossbreeding products of various cattle breeds. They did not use the knowledge about the aurochs that was available at the time, nor did they take advantage of the breeding techniques others were using to create new cattle breeds in the same period. (Van Vuure, 2005, p. 366)<br><strong>Poor engineering design methodology</strong></p>
</blockquote>
</li>
<li>With regard to directed evolution, &#39;the creativity of synthetic biology itself continues to rely on processes whose efficacy may be inscrutable to the designers themselves, and which have much in common with familiar breeding techniques&#39;</li>
<li>Irrational design in &#39;evolutionary electronics&#39;<ul>
<li>An FPGA was altered slightly and continuously tested for an ability to do a task and the best design could never have been achieved using normal design techniques<blockquote>
<p>[These cells] cannot be clamped [i.e. returned to their default b blankb  state] without degrading performance, even though there is no connected path by which they could influence the output b &amp; They must be influencing the rest of the circuit by some means other than the normal cell-to-cell wires: this probably takes the form of a very localised interaction with immediately neighbouring components. Possible mechanisms include interaction through the power-supply wiring, or electromagnetic coupling. ( Thompson, 2002)</p>
</blockquote>
</li>
</ul>
</li>
<li>Reducible complexity is not an artifact of evolution (T. Lewens 2004), therefore we should not expect to isolate components and then stick them together in a different order and expect rational behavior<blockquote>
<p>Evolution has been free to explore the full repertoire of behaviours available from the silicon resources provided, even being able to exploit the subtle interactions between adjacent components that are not directly connected. (Thompson, 2002)</p>
</blockquote>
</li>
<li><strong>Does natural selection prefer modularity? Will it pick a modular design vs a non-modular design?</strong><blockquote>
<p>It remains unclear whether natural selection is able to favour modularity<br>Natural selection is not constrained by a prior methodological commitment to rationality, and since natural selection must build traits that are effective in the face of regular internal and external insults, natural selection also needs to ensure that its products are robust against a range of environmental perturbations</p>
</blockquote>
</li>
<li>Abandon the idea of modularity in synthetic biology?<blockquote>
<p>Biological complexity is here acknowledged, the barriers it presents to human intervention are explicitly admitted, and attempts are made to avoid it (in the first instance, at least) via the creation of simpler systems.</p>
</blockquote>
</li>
<li>Citing M. O&#39;Malley &#39;Exploration, iterativity and kludging in synthetic biology&#39;: <blockquote>
<p> Rather than focusing on hypotheses, testing and inference, synthetic biology invites attention to the practices of exploration, iterativity and kludging. Examining such strategies in relation to synthetic biology offers new avenues of insight for philosophy of science and biology.</p>
</blockquote>
</li>
<li><strong>Why not design with system complexity in mind?</strong></li>
</ul>
</li>
</ul>

        </div>
    </body>
</html>
